database_docs/doc_base.json

```diff
@@ -1,404 +1,404 @@
+    "scheduled_off_control": "Scheduled exterior lighting off control; REQ means required; At least one ADD1 when present shall be implemented; At least one ADD2 when present shall be implemented",
+    "scheduled_shutoff": "Scheduled exterior lighting shutoff control; REQ means required; At least one ADD1 when present shall be implemented; At least one ADD2 when present shall be implemented",
+    "sensible_energy_recovery_ratio": "Sensible energy recovery ratio (SERR)",
+    "skylight_framing": "Skylight framing construction",
+    "solar_absorptance": "Solar absorptance",
+    "solar_diffusing": "Solar diffusing",
+    "solar_heat_gain_coefficient": "Solar heat gain coefficient",
+    "solar_transmittance_at_normal_incidence": "Solar transmittance at normal incidence",
+    "space_type_name": "Space type name",
+    "specialty_lighting_schedule": "Schedule to use for specialty lighting",
+    "specific_heat": "Specific heat (Btu/lb.F)",
+    "standard_model": "Computer room air conditioner unit type",
+    "standards_construction_type": "Reference to typical construction",
+    "standby_loss_base": "Standby loss base allowance coefficient, the standby loss requirement is calculated as follows: standby_loss_base + capacity * standby_loss_capacity_allowance or standby_loss_base + (volume * standby_loss_square_root_volume_allowance) ^ 0.5 or standby_loss_base + volume * standby_loss_volume_allowance",
+    "standby_loss_capacity_allowance": "Standby loss capacity allowance coefficient, the standby loss requirement is calculated as follows: standby_loss_base + capacity * standby_loss_capacity_allowance",
+    "standby_loss_square_root_volume_allowance": "Standby loss square root volume allowance coefficient, the standby loss requirement is calculated as follows: standby_loss_base + (volume * standby_loss_square_root_volume_allowance) ^ 0.5",
+    "standby_loss_volume_allowance": "Standby loss volume allowance coefficient, the standby loss requirement is calculated as follows: standby_loss_base + volume * standby_loss_volume_allowance",
+    "standby_mode_power": "Standby mode power (W)",
+    "start_date": "Requirement applicability start date",
+    "subcategory": "System subcategory",
+    "subtype": "Furnace subtype",
+    "supplemental_lighting_power_density": "Supplemental lighting power density",
+    "supplemental_lighting_power_density_unit": "Supplemental lighting power density unit",
+    "support_electric_equipment_space_type_name_tags": "Support electric equipment space type name tags",
+    "synchronous_speed": "Motor synchronous speed (RPM)",
+    "systems_11_12_13_minimum_threshold": "@todo",
+    "systems_11_12_13_minimum_threshold_descriptor": "@todo",
+    "systems_3_4_minimum_threshold": "@todo",
+    "systems_3_4_minimum_threshold_descriptor": "@todo",
+    "systems_5_through_8_minimum_threshold": "@todo",
+    "systems_5_through_8_minimum_threshold_descriptor": "@todo",
```

Comment:
@AmmarDeh to add descriptions.

Reply:
"systems_11_12_13_minimum_threshold": "Minimum threshold requirements at which economizers are included for baseline systems 11, 12 and 13",
    "systems_11_12_13_minimum_threshold_descriptor": "Descriptor type",
    "systems_3_4_minimum_threshold": "Minimum threshold requirements at which economizers are included for baseline systems 3 and 4",
    "systems_3_4_minimum_threshold_descriptor": "Descriptor type",
    "systems_5_through_8_minimum_threshold": "Minimum threshold requirements at which economizers are included for baseline systems 5 through 8",
    "systems_5_through_8_minimum_threshold_descriptor": "Descriptor type",

Reply:
Thanks!